### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by rizumita**

### CODE REVIEW

File: SelmSample/HistoryView.swift [hunk 1]
```diff
@@ -22,13 +22,19 @@ struct HistoryView : View, SelmView {
                 }.onDelete(perform: dispatch • Msg.remove)
             }
         }
+        .onAppear {
+            self.store.subscribe()
```
Comment (kylelol) [inline SelmSample/HistoryView.swift:26]: Do we do this to prevent the view from updating when its not on screen?

Comment (rizumita) [reply SelmSample/HistoryView.swift:26]: @kylelol Yes.

Comment (kylelol) [reply SelmSample/HistoryView.swift:26]: 👍  I wonder if we can try to lift that functionality into the library, so we don't have to remember to do this all the time. Thoughts?

Comment (rizumita) [reply SelmSample/HistoryView.swift:26]: @kylelol 
Do you have a good idea?

Comment (kylelol) [reply SelmSample/HistoryView.swift:26]: I'll create a PR with something, and we can discuss it there 👍

Comment (rizumita) [reply SelmSample/HistoryView.swift:26]: @kylelol Thank you!

**Pull request merged**